packages/x-charts/src/ChartsTooltip/ChartsTooltipTable.ts

```diff
@@ -27,10 +27,10 @@ export const ChartsTooltipRow = styled('tr', {
   name: 'MuiChartsTooltip',
   slot: 'Row',
 })(({ theme }) => ({
-  'tr:first-of-type& td': {
+  '& tr:first-of-type& td': {
```

Comment:
Shouldn't there be no space here to correctly target what we expect? 🤔 
```suggestion
  '&tr:first-of-type& td': {
```

Reply:
Ah wait, I missed the &